tests/test_db.rs

```diff
@@ -163,6 +165,45 @@ fn snapshot_test() {
     }
 }
 
+#[derive(Clone)]
```

Comment:
#[cfg(test)] ??? We don't need a `SnapshotWrapper` in a production code, do we ?

Reply:
`SnapshotWrapper` is in the tests directory, therefore, it is not visible in the main crate.

Reply:
Oh. It's ok then.